iocage/cli/stop.py

```diff
@@ -35,21 +36,21 @@
               help="Will stop all jails with boot=on, in the specified"
                    " order with higher value for priority stopping first.")
 @click.argument("jails", nargs=-1)
-def cli(rc, jails):
+@click.option("--log-level", "-d", default=None)
+def cli(rc, jails, log_level):
     """
     Looks for the jail supplied and passes the uuid, path and configuration
     location to stop_jail.
     """
-    if not jails and not rc:
-        ioc_common.logit({
-            "level"  : "ERROR",
-            "message": 'Usage: iocage stop [OPTIONS] JAILS...\n'
-                       '\nError: Missing argument "jails".'
-        })
-        exit(1)
+    logger = iocage.lib.Logger.Logger(print_level=log_level)
+    ioc_jails = iocage.lib.Jails.Jails(logger=logger)
+
+    for jail in ioc_jails.list(filters=jails):
+        logger.log(f"Stopping jail {jail.humanreadable_name}")
+        try:
+            jail.stop()
+        except:
```

Comment:
Dont we want to print to the user why stop failed?

Also maybe handle KeyboardInterrupt specially, to skip current "stop"?